docs/blog.md

```diff
@@ -2,30 +2,30 @@
 
 Web developers and web standards authors alike strive to live up to the promise of "universality" - the idea that the web _should_ be available to all. This concept drives many innovations in web technology, as well as being fundamentally built in to the philosophy of the open standards on which the web is based.
 
-In order to achieve this, we find that having some carefully chosen user context allows web developers to create more flexible and user experiences. We think it's time to expand our view of user context to include __modality__ - but before we flesh that out, let's clarify what we mean by "user context".
+In order to achieve this, we frequently find that having some carefully chosen information about how the user intends to view the content (a concept called "user context") allows web developers to create more flexible and useful user experiences. In this post, we'll lay out a case that it's time to expand our view of user context to include the concept of __modality__ (how the user is physically interacting), but before we flesh that out, let's take a look at "user context".
```

Comment:
s/(how the user is physically interacting),/- _how_ the user is interacting with the page -/